malduck/crypto/des3.py

```diff
@@ -18,9 +18,33 @@ def _get_cipher(self, key, iv):
         return DES3Cipher.new(key, DES3Cipher.MODE_CBC, iv=iv)
 
     def encrypt(self, key, iv, data):
+        """
+        Encrypts buffer using DES/DES3 algorithm in CBC mode.
+
+        :param key: Cryptographic key (8 bytes for single DES, 16 or 24 bytes)
```

Comment:
Would we maybe want to switch the order here?
So: `(16 or 24 bytes, 8 bytes for single DES)`
It would suggest that the cipher behaves normally for 16 and 24 bytes and there is some kind of fallback for 8 bytes.